### ISSUE TITLE
Set_A.txt and Set_B.txt in ORIGA dataset

### ISSUE THREAD

**Issue by baaaohaaan** [Guzaiwang/CE-Net#18]
Hi! Could you please provide Set_A.txt and Set_B.txt for experiment on ORIGA dataset? I searched online for how ORIGA dataset was divided into training set and test set, but found nothing. According to your data.py, Set_A.txt and Set_B.txt are used to divide ORIGA dataset, so I would be appreciated if you could provide them. Thx!

**Comment by meigeni0929** [comment]
我也有这个问题 这个是为什么呢

**Comment by 594Surefour** [comment]
i have the same question

**Comment by Swiffany** [comment]
i have the same problem,anyone solve it?
